provisioning/atat_provisioning.yaml

```diff
@@ -1,21 +1,48 @@
 openapi: 3.0.2
 info:
   description: >-
-    This is a DRAFT version of the API meant for provisioning resources into a JWCC Provider Cloud Environment. It will be
-    implemented and maintained by all participating CSPs.
+    This API is meant for provisioning resources in a JWCC Provider Cloud Environment.
+    The cloud products and services used in JWCC are provided by commercial cloud service providers (CSP).
+    This API will be implemented and maintained by all participating CSPs.
+
+    ### Organization of provisioned resources
+    
+    Provisioned resources under the purview of a mission owner are organized in a three-level hierarchy.
+
+    * Portfolio
+      * Application
+        * Environment
+
+    *Portfolios* consist of one or more applications. *Applications* consist of one or more environments.
+    *Environments* directly contain provisioned resources.
```

Comment:
```suggestion
    *Environments* directly contain provisioned resources.
    Although the implementation of these concepts will vary by vendor, the following principles are expected to be adhered to:
    - Administrative access to an Application should provide inherited access to all contained Environments
    - Administrative access to a Portfolio should provide inherited access to all contained Application
    - Each Application under a given Portfolio will have isolated security controls from one another
    - Each Environment under a given Application will have isolated security controls from one another
    - Actual CSP resources & services (any IaaS or Paas offerings from the CSP) should only be procured in an Environment
```

Reply:
Added in b65917de3a6cb37d94e511d9fc97efa7ddc62b66